graphql-ballerina/build.gradle

```diff
@@ -147,8 +149,9 @@ task initializeVariables {
             testParams = "--skip-tests"
         }
 
-        if (graph.hasTask(":${packageName}-ballerina:publish")) {
+        if (graph.hasTask(":${packageName}-ballerina:publish") || graph.hasTask(":${packageName}-ballerina:publishToMavenLocal")) {
```

Comment:
According to this logic, let say, someone has configured the ballerina central token. Then during the local publishToMavenLocal, the module will publish to central. What is the reason for doing that?.

Reply:
We added this to enable publishing locally, when they have configured the access token. (Usually we don't). But I see your point.

Will come up with a solution.

Reply:
As you all discussed, I think we don't need to be able to publish to central from local machine. It is always good to do it via github actions.

Reply:
Fixed with #89